test/test_geocaching.py

```diff
@@ -28,15 +28,15 @@ def test_my_dnfs(self):
 
     def test_search(self):
         with self.subTest("normal"):
-            tolerance = 2
-            expected = {"GC5VJ0P", "GC41FJC", "GC50AQ6", "GC167Y7", "GC7RR74", "GC167Y7"}
+            tolerance = 3
```

Comment:
Do we have some way to avoid this tolerance value to further grow?

Reply:
Yup, resolved in https://github.com/tomasbedrich/pycaching/pull/206/commits/8534729f50fae8825939683d2410606ac22ecae2